src/Rooms.java

```diff
@@ -0,0 +1,15 @@
+public enum Rooms {
+    A123(new RoomHour(8, 16)),
+    B123(new RoomHour(9, 17)),
```

Comment:
It could also do without the RoomHour class -> you could just use 2 int variables as Rooms enum variables -> but that is fine too ;)